controllers/routemonitor/routemonitor_supplement.go

```diff
@@ -4,34 +4,34 @@ import (
 	"context"
 
 	"github.com/openshift/route-monitor-operator/api/v1alpha1"
+	"github.com/openshift/route-monitor-operator/pkg/const/blackbox"
+	"github.com/openshift/route-monitor-operator/pkg/util/reconcile"
 	utilreconcile "github.com/openshift/route-monitor-operator/pkg/util/reconcile"
 )
 
-func (r *RouteMonitorReconciler) EnsureBlackBoxExporterResourcesExists(ctx context.Context) error {
-	if err := r.EnsureBlackBoxExporterDeploymentExists(ctx); err != nil {
-		return err
-	}
-	// Creating Service after because:
-	//
-	// A Service should not point to an empty target (Deployment)
-	if err := r.EnsureBlackBoxExporterServiceExists(ctx); err != nil {
-		return err
-	}
-	return nil
+//go:generate mockgen -source $GOFILE -destination ../../pkg/util/test/generated/mocks/$GOPACKAGE/blackboxexporter.go -package $GOPACKAGE BlackboxExporter
+type BlackboxExporter interface {
```

Comment:
why isn't that in the interface file?

Reply:
See comment about constant. Interfaces should stay with the user.

Reply:
oh, so you are saying the interface file I did there should be merged with the supplement?
I am totally fine with that

Reply:
I don't think we need necessarily need to change that, as it lives in a different package and is its own controller. So it can have a slightly different design.

Reply:
okay.. not sure how I feel about it but we can close if this is a blocker